charts/risingwave/values.yaml

```diff
@@ -974,6 +974,13 @@ frontendComponent:
   ##
   embeddedServing: false
 
+  ## @param autoDeregistration Auto-deregistration of frontend pods. If enabled, frontend pods will be
```

Comment:
Make it clear that the value only takes effect when `embeddedServing` is turned on.